tests/local-integration-test.sh

```diff
@@ -72,29 +72,25 @@ workspace:
   path: src/github.com/vmware/vic
 
 pipeline:
-  clone:
-    image: plugins/git
-    tags: true
-    # dont clone submodules
-    recursive: false
   vic-integration-test-on-pr:
     image: gcr.io/eminent-nation-87317/vic-integration-test:1.46
     pull: true
     environment:
       GITHUB_AUTOMATION_API_KEY: $GITHUB_TOKEN
-      TEST_URL_ARRAY:   $(govc env -x GOVC_URL_HOST)
-      TEST_USERNAME:    $(govc env GOVC_USERNAME)
-      TEST_PASSWORD:    $(govc env GOVC_PASSWORD)
-      TEST_DATASTORE:   ${GOVC_DATASTORE:-$(basename "$(govc ls datastore | head -1)")}
-      TEST_RESOURCE:    ${GOVC_RESOURCE_POOL:-$(govc ls host/*/Resources)}
-      BRIDGE_NETWORK:   $BRIDGE_NETWORK
-      PUBLIC_NETWORK:   $PUBLIC_NETWORK
-      DOMAIN:           $DOMAIN
+      TEST_URL_ARRAY:   "$(govc env -x GOVC_URL_HOST)"
```

Comment:
Did you try this after settings these secrets? Asking because secrets format has changed in latest drone version, pls refer to `.drone.local.yml`